src/content/docs/apm/agents/go-agent/configuration/distributed-tracing-go-agent.mdx

```diff
@@ -185,138 +187,20 @@ Infinite Tracing configuration settings include the standard distributed tracing
   </Collapser>
 </CollapserGroup>
 
-#### Step 4. (Optional) Customize Infinite Tracing [#customize-inf-tracing]
+<Callout variant="tip">
+  If you need help with proxy configuration, see [Proxy support](/docs/understand-dependencies/distributed-tracing/other-requirements/infinite-tracing-proxy-support).
+</Callout>
+  </Step>
+  <Step>
+### (Optional) Customize Infinite Tracing [#customize-inf-tracing]
 
 After you add the agent configuration settings, you should start seeing data in the New Relic UI. After you spend some time analyzing your data, you may want to adjust some of the features of Infinite Tracing:
```

Comment:
@nbaenam since this link and other information is in the standard distributed tracing setup, I'm going to assume that users would have already learned about how to use the UI.